packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -24,6 +25,12 @@ export enum KeyboardSpinDirection {
 }
 
 export interface ISpinButtonState {
+
+  /**
+   * Is true when the control has focus.
+   */
+  isFocused?: boolean;
```

Comment:
Is this bad ? 
The reason I added this field to the state is because we need to style the root element of the component based on the input's (a child of root) focus state. This is not possible directly through CSS - the best you can do is style the sibling elements.